systems/plants/RigidBody.m

```diff
@@ -247,15 +247,13 @@ function dofnum(obj)
 
     function body = makeBelongToCollisionFilterGroup(body,collision_fg_id)
       for id = reshape(collision_fg_id,1,[])
-        body.collision_filter.belongs_to = ...
-          bitor(body.collision_filter.belongs_to,bitshift(1,id-1));
+        body.collision_filter.belongs_to(collision_fg_id) = true;
```

Comment:
out of curiousity, why the change away from the bit logic?  might we be introducing new bugs here?